src/components/Modal/Votes/VotesList.css

```diff
@@ -1,65 +1,105 @@
-.VotesList .VotesList_Container_Header {
+.VotesList .VotesList__HeaderContainer {
   margin: 0 auto 20px;
 }
 
-.VotesList .VotesList_Container_Items {
+.VotesList .VotesList__ItemsContainer {
   margin: 0 auto 40px;
   height: 50vh;
   padding: 20px 0;
   overflow-y: auto;
   overflow-x: hidden;
 }
 
-.VotesList .VotesList_Container_Items .VoteList_Item:hover {
+.VotesList .VotesList__ItemsContainer .VoteList__Item:hover {
   cursor: pointer;
 }
 
-.VotesList .VotesList_Container_Header .VotesList_Header {
+.VotesList .VotesList__HeaderContainer .VotesList__Header {
   color: #a4a2a8;
 }
 
-.VotesList_Divider {
+.VotesList__Divider {
   position: relative;
 }
 
-.VotesList_Container_Items
-  .VotesList_Divider
-  .VotesList_Divider_Line:first-child:after {
-  content: "";
+.VotesList__ItemsContainer .VotesList__Divider .VotesList__DividerLine:first-child:after {
+  content: '';
   position: absolute;
   top: 0;
   width: 95%;
-  border-bottom: 1px solid #e7e6e9;
+  border-bottom: 1px solid var(--black-300);
 }
 
-.VotesList_Divider .VotesList_Divider_Line:before {
-  content: "";
+.VotesList__Divider .VotesList__DividerLine:before {
+  content: '';
   position: absolute;
   bottom: 0;
   width: 95%;
-  border-bottom: 1px solid #e7e6e9;
+  border-bottom: 1px solid var(--black-300);
+}
+.VoteList__Item:has(+ .VoteList__ShowButtonContainer)::before,
+.VoteList__Item:has(+ .VoteList__LowQualityDividerContainer)::before {
+  border-bottom: none;
 }
 
 .VotesList .dg.dcl-avatar {
-  margin: 0 .25em 0 0;
+  margin: 0 0.25em 0 0;
 }
 
-.row.VoteList_Item.VotesList_Divider_Line {
+.row.VoteList__Item.VotesList__DividerLine {
   display: flex;
   align-items: baseline;
 }
 
+.row.VoteList__Item.VoteList__Item--lowQualityActive {
```

Comment:
We are using "kebab case" for css modifiers names as a pattern (--low-quality-active)